### PR TITLE
Arpeggiator proof-of-concept 

### PR DESCRIPTION
**Pull request by rileynwong**

### CODE REVIEW

File: MyProjects/arp/.arp.cpp.swp
Comment (isaacpearl) [inline MyProjects/arp/.arp.cpp.swp:1]: add `*.swp` to git ignore

File: seed/Button/Button.cpp
Comment (isaacpearl) [inline seed/Button/Button.cpp:1]: Do we want to commit these changes to an official example?

**Review by isaacpearl — CHANGES_REQUESTED** [review]

**Review by isaacpearl — CHANGES_REQUESTED** [review]
See comments

File: seed/Button/Button.cpp
Comment (rileynwong) [reply seed/Button/Button.cpp:1]: Ah good catch!

**Review by isaacpearl — APPROVED** [review]

**Pull request merged**